extensions/markdown-language-features/package.json

```diff
@@ -121,6 +121,14 @@
       }
     ],
     "commands": [
+      {
+        "command": "_markdown.copyImage",
+        "title": "%markdown.copyImage.title%",
+        "icon": {
```

Comment:
This command does not need an icon. The icons are only shown if the command is used in a toolbar